upstairs/src/block_req.rs

```diff
@@ -65,7 +74,10 @@ impl BlockReqWaiter {
     /// Consume this BlockReqWaiter and wait on the message
     ///
     /// If the other side of the oneshot drops without a reply, log an error
-    pub async fn wait(self, log: &Logger) -> Result<(), CrucibleError> {
+    pub async fn wait(
```

Comment:
Annotate with `#[must_use]`, to force people to think about the buffer (or assert that it is none)?

Reply:
I like this suggestion, but this didn't seem to have any effect. However, adding the `#[must_use]` to `Buffer` in 115510f did :)